StripeConnect/StripeConnectTests/Internal/Webview/MessageHandlers/OnSetterFunctionCalledMessageHandlerTests.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  OnSetterFunctionCalledMessageHandlerTests.swift
+//  StripeConnectTests
+//
+//  Created by Chris Mays on 9/23/24.
+//
+
+@testable import StripeConnect
+import XCTest
+
+class OnSetterFunctionCalledMessageHandlerTests: ScriptWebTestBase {
```

Comment:
Optional –
Should we include some tests to ensure all the handlers are called when there's multiple, etc?

There's also a good amount of coverage in the specific handler tests (e.g. `OnExitMessageHandlerTests`), so not critical

Reply:
I'll follow up with this.